Create my-first-week-as-a-trainee-teacher (#1754)
DFE-Digital/get-into-teaching-app: app/views/content/blog/my-first-week-as-a-trainee-teacher.md

---
title: My first week as a trainee teacher
author: Cindy Ventura
date: "2021-09-17"
images:
  cindy-ventura:
    path: "media/images/content/blog/cindy-ventura.jpg"
    alt: "A photograph of trainee teacher Cindy Ventura"
description: |-
  Cindy recently started the Postgraduate Diploma in Education (PGDE) teacher training course at Sheffield University. She kept a diary during her first week on the course and shares her experiences here.
keywords:
  - teacher training
  - trainee teacher course
  - university-led teacher training
tags:
  - becoming a teacher
---

$cindy-ventura$

Cindy recently started the Postgraduate Diploma in Education (PGDE) teacher training course at Sheffield University. She kept a diary during her first week on the course and shares her experiences here.

> 

Reviewer: Is it worth putting a subheading here to break up the intro from Cindy's words?

Author: How about Cindy's diary?

Reviewer 2: Good call. On other posts we make the final paragraph italic so it's clear it's separate.

I created [a ticket on trello](https://trello.com/c/sqqWfKfn/1701-improve-blog-footers) on the subject of improving the footers in general.

> Throughout my time at university, I knew I wanted a career that would make a difference and give me opportunities for lifelong learning. I was interested in languages and worked as a language assistant in different schools in Sheffield, which developed my interest in teaching. I applied for the PGDE program at Sheffield University and since my interview in March, the time has passed so fast that I now find myself preparing my backpack for my first day.

Reviewer: programme rather than program?

> Throughout my time at university, I knew I wanted a career that would make a difference and give me opportunities for lifelong learning. I was interested in languages and worked as a language assistant in different schools in Sheffield, which developed my interest in teaching. I applied for the PGDE program at Sheffield University and since my interview in March, the time has passed so fast that I now find myself preparing my backpack for my first day.

Reviewer: ```suggestion
Throughout my time at university, I knew I wanted a career that would make a difference and give me opportunities for lifelong learning. I was interested in languages and worked as a language assistant in different schools in Sheffield, which developed my interest in teaching. I applied for the PGDE programme at Sheffield University and since my interview in March, the time has passed so fast that I now find myself preparing my backpack for my first day.
```

## Monday

> I start at 9:30am with an introductory lecture to the entire PGDE cohort, where further information about the course is given; the assessments and societies that we can join.

Reviewer: ```suggestion
I start at 9:30am with an introductory lecture to the entire PGDE cohort, where further information is given about the course, the assessments and societies that we can join.
```

After lunch, the PGDE course leader has organised ice breakers to introduce the students to each other. I enjoy this informal activity as it helps me to get to know my colleagues better, but also to understand what their reasons are for becoming teachers.

Afterwards, I continue my day with two seminars about successfully teaching languages and classroom materials, including games that could be used to teach and engage students with their learning.

I enjoy my first day because I meet many different people from different PGDE subjects, but I also become more familiar with the requirements of the course.

## Tuesday

> My day begins with a lecture on teaching skills and the different course programs available for teachers’ development, such as computing. We also hear a skills and job application talk by the recruitment company Hays. They present us with information on job applications and how to write a thriving, supportive statement for our CVs. They also share the different platforms where teaching jobs can be found and talk to us about the job market for teachers.

Reviewer: ```suggestion
My day begins with a lecture on teaching skills and the different course programmes available for teachers’ development, such as computing. We also hear a skills and job application talk by the recruitment company Hays. They present us with information on job applications and how to write a thriving, supportive statement for our CVs. They also share the different platforms where teaching jobs can be found and talk to us about the job market for teachers.
```

I enjoy the variety of information given during the morning events as it increases my understanding of the abilities and skills that schools are looking for when employing teachers.

> In the afternoon, I attend two subject seminars on the History of the British curriculum, which explain the differences and similarities between European and British secondary education.

Reviewer: ```suggestion
In the afternoon, I attend two subject seminars on the history of the British curriculum, which explain the differences and similarities between European and British secondary education.
```

## Wednesday

Today I attend several seminars on the practical side of teaching, such as marking, how to find sources and papers to prepare students for exams and techniques, how to adapt resources for different key stages and how to plan lessons according to the curriculum.

I enjoy these seminars because they are more practical and give me the chance to apply and put into practice what I already know, as well as learn more about the curriculum of the different key stages.

## Thursday

This morning, all the students enrolled on the course attend an informative lecture on the different master components that are part of the PGDE, as well as information about school placements.

> Similar to previous days, my lessons continue in the afternoon with a session focused on preparing us for the group master assignment. We are divided into small groups of four and asked to consult a newspaper article focused on the topic: ’issues in education’. We then work together on a 10-minute presentation to deliver to the rest of the class.

Reviewer: ```suggestion
Similar to previous days, my lessons continue in the afternoon with a session focused on preparing us for the group master assignment. We are divided into small groups of four and asked to consult a newspaper article focused on the topic 'issues in education'. We then work together on a 10-minute presentation to deliver to the rest of the class.
```

I don’t enjoy this activity because I am nervous about presenting in front of so many people, especially on a topic that I don’t know much about. However, even though I am nervous, the presentation goes well, and I overcome the challenge with the support of my group, who encourage me throughout the presentation.

Reflecting on this experience, I admit that this practice presentation was a good way to prepare for my upcoming school placements where I will be presenting and leading lessons.

## Friday

> My day starts differently to previous days. I spend the morning at home reading an article and watching a pre-recorded seminar on children and poverty. I then read some papers and watch two short documentaries on education and poverty. After completing these tasks, I met my subject group on Google Meet and we spent the rest of the morning discussing what we have seen.

Reviewer: ```suggestion
My day starts differently to previous days. I spend the morning at home reading an article and watching a pre-recorded seminar on children and poverty. I then read some papers and watch two short documentaries on education and poverty. After completing these tasks, I meet my subject group on Google Meet and we spend the rest of the morning discussing what we have seen.
```

> My day starts differently to previous days. I spend the morning at home reading an article and watching a pre-recorded seminar on children and poverty. I then read some papers and watch two short documentaries on education and poverty. After completing these tasks, I meet my subject group on Google Meet and we spend the rest of the morning discussing what we have seen.

Reviewer 2: ```suggestion
My day starts differently to previous days. I spend the morning at home reading an article and watching a pre-recorded seminar on children and poverty. I then read some papers and watch two short documentaries on education and poverty. After completing these tasks, I met my subject group on Google Meet and we spent the rest of the morning discussing what we have seen.
```

> In the afternoon I attend a seminar on GCSE and A-level assessment. In this session I work in a group and identify and discuss the different assessment specifications for languages.

Reviewer 2: ```suggestion
In the afternoon I attended a seminar on GCSE and A-level assessment. In this session we worked in a group to identify and discuss the different assessment specifications for languages.
```

> In the afternoon I attended a seminar on GCSE and A-level assessment. In this session we worked in a group to identify and discuss the different assessment specifications for languages.

Reviewer: ```suggestion
In the afternoon I attend a seminar on GCSE and A-level assessment. In this session we work in a group to identify and discuss the different assessment specifications for languages.
```

> The group work is important because it aimed to familiarise us with the different subjects’ language specifications and syllabus, particularly interesting for me as I didn’t attend school in the UK, so learning about specifications and the curriculum was exciting to me.

Reviewer: ```suggestion
The group work is important because it aimed to familiarise us with the different subjects’ language specifications and syllabus, particularly interesting for me as I didn’t attend school in the UK, so learning about specifications and the curriculum is exciting to me.
```

> 

Reviewer: Worth putting another sub-heading here to distinguish Friday from the summary (change in tense)?

Author: Agreed but I can't think of one! 'Final thoughts'

Appreciate your eyes on this Gemma - English isn't her first language so it took a lot of editing from the original!

> My first week was hectic, which I didn’t expect, but it gave me many things to reflect upon and I learned a lot – it enabled me to see the importance and the challenges of education from different sides. In addition, I met new people from other countries and learned their views and approaches on teaching and learning, which was very enriching.

Reviewer: Worth putting another sub-heading here to distinguish Friday from the summary (change in tense)?

> If you’d like to be in Cindy’s shoes this time next year, [find out more about your next steps](/steps-to-become-a-teacher).

Reviewer: ```suggestion
If you’d like to be in Cindy’s shoes this time next year, [find out more about your next steps to becoming a teacher](/steps-to-become-a-teacher).
```

> If you’d like to be in Cindy’s shoes this time next year, [find out more about your next steps](/steps-to-become-a-teacher).

Reviewer: would perhaps suggest not putting 'this time next year' as it will date

Author: How about just deleting that so just 'If you'd like to be in Cindy's shoes, find out more.....   Sorry I can't work out how to change it myself!

---
title: My first week as a trainee teacher
author: Cindy Ventura
date: "2021-09-20"
images:
  cindy-ventura:
    path: "media/images/content/blog/cindy-ventura.jpg"

>     alt: "A photograph of trainee teacher Cindy Ventura"

Reviewer 2: @gemmadallmandfe - any hints on improving this alt text?

Reviewer: Alt text best practice is not to start with 'A photograph of'

How about:

Trainee teacher Cindy Ventura wearing a red polo shirt

(or you could just put 'Trainee teacher Cindy Ventura')
